terraform/helm/aptos-node/files/haproxy.cfg

```diff
@@ -1,47 +1,114 @@
 global
```

Comment:
This file feels very similar to the other one, is there some way to deduplicate?

Reply:
No real way to de-duplicate since they work with different backends, which are dependent on the deployment environment. What we can do is remove as many template points as possible in this file, and just hardcode all these values, e.g. 

```
tune.rcvbuf.client {{ $.Values.haproxy.limits.validator.tcpBufSize }}
# becomes
tune.rcvbuf.client 4194304
```

That way the file configurations are exactly the same other than the backend /server blocks
cc @Markuze

Reply:
They are currently the same, but they may diverge in the future.